tests/package-title-dir-rtl-001/OPS/package.opf

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<package xmlns="http://www.idpf.org/2007/opf" xmlns:epub="http://www.idpf.org/2007/ops" version="3.0" xml:lang="en" unique-identifier="pub-id">
+    <metadata xmlns:dc="http://purl.org/dc/elements/1.1/">
+        <dc:coverage>Internationalization</dc:coverage>
+        <dc:creator>Ivan Herman</dc:creator>
+        <dc:date>2021-09-22</dc:date>
+        <dc:description>The 'dc:title' element contains a text whose proper rendering requires bidi control. The element's 'dir' attribute is set to 'rtl'; the title should display correctly.</dc:description>
+        <dc:identifier id="pub-id">package-title-dir-rtl-001</dc:identifier>
+        <dc:language>en</dc:language>
+        <dc:title dir="rtl" xml:lang="he">CSS: הרפתקה חדשה!</dc:title>
+        <meta property="dcterms:alternative">Correct 'dir' on the title</meta>
```

Comment:
Since there should be several dir tests (right?), should this say "Correct rendering of 'dir="rtl"' on the title"? I think you will need an ltr case later, an auto value, an implicit auto, etc.

Reply:
+1. Similar changes on the titles of all the others, too.

Reply:
Done it in https://github.com/w3c/epub-tests/pull/31/commits/dc9f0d396bd4d814812b3d707225bef4f0f59357